core/src/core-plugin-definitions.ts

```diff
@@ -1612,6 +1612,9 @@ export interface ToastPlugin extends Plugin {
 
 export interface ToastShowOptions {
   text: string;
+    /**
```

Comment:
Can you align the * in the comments?

```suggestion
  /**
```